snap.go

```diff
@@ -35,12 +35,17 @@ import (
 
 var sha3_384oid = asn1.ObjectIdentifier{2, 16, 840, 1, 101, 3, 4, 2, 9}
 
+// ClassicModelGradeMask is ORed with the model grade code when
+// measuring a classic snap model.
+const ClassicModelGradeMask uint32 = 0x80000000
```

Comment:
As a follow-up, maybe it could be good to leave a comment in https://github.com/snapcore/snapd/blob/2.57.2/asserts/model.go#L367 mentioning that now the highest bit is reserved as it is used in the model HMAC for differentiating classic/core?

Reply:
yes, I plan to do something for that